msteams-platform/tabs/how-to/authentication/auth-silent-aad.md

```diff
@@ -112,3 +112,14 @@ if (authContext.isCallback(window.location.hash)) {
     }
 }
 ```
+
+### Handle sign out flow
+
+Use the following code to handle sign out flow in AAD Auth:
+
+```javascript
+function logoutV1() {
+localStorage.clear();
+window.location.href = "@Url.Action("ChooseAuth", "Home")";
```

Comment:
Please replace chooseAuth with `<<Action Name>>` and Home to `<<Controller Name>>`